deploy/playbooks/roles/common/tasks/systemd.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+- name: ensure /var/log/celery dir exists
+  sudo: true
+  file: path=/var/log/celery state=directory owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} recurse=yes
+
+- name: ensure /etc/conf.d/ dir exists
+  sudo: true
+  file: path=/etc/conf.d state=directory
```

Comment:
Typically this environment file would be `/etc/sysconfig/chacra` on RHEL, or `/etc/default/chacra` on Ubuntu